
- remove unnecessary stuff which runs httpd during build
- drop unnecessary --libs output from ap?-?-config

--- httpd-2.1.10/support/apxs.in.apxs
+++ httpd-2.1.10/support/apxs.in
@@ -184,34 +184,6 @@
     }
 }
 
-##
-##  Initial shared object support check
-##
-my $httpd = get_vars("sbindir") . "/" . get_vars("progname");
-$httpd = eval qq("$httpd");
-$httpd = eval qq("$httpd");
-my $envvars = get_vars("sbindir") . "/envvars";
-$envvars = eval qq("$envvars");
-$envvars = eval qq("$envvars");
-
-#allow apxs to be run from the source tree, before installation
-if ($0 =~ m:support/apxs$:) {
-    ($httpd = $0) =~ s:support/apxs$::;
-}
-
-unless (-x "$httpd") {
-	error("$httpd not found or not executable");
-	exit 1;
-}
-
-unless (grep /mod_so/, `. $envvars && $httpd -l`) {
-    error("Sorry, no shared object support for Apache");
-    error("available under your platform. Make sure");
-    error("the Apache module mod_so is compiled into");
-    error("your server binary `$httpd'.");
-    exit 1;
-}
-
 sub get_config_vars{
     my ($file, $rh_config) = @_;
 
@@ -433,9 +405,9 @@
 
     if ($opt_p == 1) {
         
-        my $apr_libs=`$apr_config --cflags --ldflags --link-libtool --libs`;
+        my $apr_libs=`$apr_config --cflags --ldflags --link-libtool`;
         chomp($apr_libs);
-        my $apu_libs=`$apu_config --ldflags --link-libtool --libs`;
+        my $apu_libs=`$apu_config --ldflags --link-libtool`;
         chomp($apu_libs);
         
         $opt .= " ".$apu_libs." ".$apr_libs;
